hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -377,6 +377,42 @@ spec = parallel $ do
 
                 waitUntil [n1] $ GetUTxOResponse testHeadId (utxoRefs [2, 42])
 
+      it "can request decommit" $
```

Comment:
i think this test is redundant as it is being covered by the below.

wdyt?

Reply:
I think so too so we could remove it.